### ISSUE TITLE
Hyper-parameters for training

### ISSUE THREAD

**Issue by ntuyt** [naver-ai/pit#2]
Dear authors,
Just wonder the detailed hyper-parameters for training?
Can I just use the hyper-parameters for training deit for training pit?

**Comment by bhheo** [comment]
Hi

Yes.  We trained pit with hyper-parameters of deit.

But, there are a few differences
- We replace AdamW with AdamP (https://github.com/clovaai/AdamP). But, no changes on optim parameters.
- `--repeated_aug` is only used for `PiT-B` and other networks are trained with `--no_repeated_aug`.
- We doubled warmup epochs `--warmup_epochs 10` for `PiT-B` 

Note that the impacts of differences are not big, so it works well with just `deit` setting without changes.

**Comment by ntuyt** [comment]
Thanks for your reply. I get it.
